DevCycleTests/Models/DevCycleClientTests.swift

```diff
@@ -5,88 +5,95 @@
 //
 
 import XCTest
-#if os(iOS) || os(tvOS)
-import UIKit
-#elseif os(watchOS)
-import WatchKit
-#endif
 
 @testable import DevCycle
 
+#if os(iOS) || os(tvOS)
+    import UIKit
+#elseif os(watchOS)
+    import WatchKit
+#endif
 
 class DevCycleClientTest: XCTestCase {
     private var service: MockService!
     private var user: DevCycleUser!
     private var builder: DevCycleClient.ClientBuilder!
     private var userConfig: UserConfig!
-    
-    override func setUp() {
-        self.service = MockService()
-        self.user = try! DevCycleUser.builder()
-                    .userId("my_user")
-                    .build()
-        self.builder = DevCycleClient.builder().service(service)
 
+    override func setUp() {
         let data = getConfigData(name: "test_config")
-        let dictionary = try! JSONSerialization.jsonObject(with: data, options: .fragmentsAllowed) as! [String:Any]
+        let dictionary =
+            try! JSONSerialization.jsonObject(with: data, options: .fragmentsAllowed)
+            as! [String: Any]
         self.userConfig = try! UserConfig(from: dictionary)
+        self.service = MockService(userConfig: self.userConfig)
```

Comment:
Update the mock service to actually set the `userConfig` to actually call the callbacks / async promises.